react/features/toolbox/components/Toolbox.native.js

```diff
@@ -252,14 +273,23 @@ class Toolbox extends Component<Props> {
                     style = { style }
                     underlayColor = { underlayColor } />
                 {
-                    _SHARE_ROOM_TOOLBAR_BUTTON
+                    _SHARE_ROOM_TOOLBAR_BUTTON && !showInviteButton
```

Comment:
@saghul, if I understand Nichole's designs right, the "share room" is going to become "copy meeting url" and the InviteButton is going to be "add/invite people"? So then the two buttons will not be mutually exclusive but they will coexist?

Reply:
Good question. TBH I don't know. The new design includes a share sheet with a bunch of options, I guess "invite participants" will be one of them, so in this regard they can be considered mutually exclusive, I think.

In the current form they definitely are, so I guess we can decide this after this lands, while we work on the redesign.